src/shared/version.go

```diff
@@ -1,5 +1,5 @@
 package shared
 
 const (
-	HyperdriveVersion string = "0.3.0-dev"
+	HyperdriveVersion string = "0.2.5-dev"
```

Comment:
If you're rolling this into `main` then I'd keep the v0.3.0 version since so much has changed on the canonical branch.